slf4j/src/main/scala/zio/logging/slf4j/Slf4jLogger.scala

```diff
@@ -75,32 +75,55 @@ object Slf4jLogger {
    * Creates a slf4j logger that puts all the annotations defined in `mdcAnnotations` in the MDC context
    */
   def makeWithAnnotationsAsMdc(
-    mdcAnnotations: List[LogAnnotation[_]],
+                                mdcAnnotations: List[LogAnnotation[_]],
+                                logFormat: (LogContext, => String) => String = (_, s) => s
+                              ): ULayer[Logging] = {
+    val annotationNames = mdcAnnotations.map(_.name).toSet
+    val filter = (renderContext: Map[String, String]) => renderContext.filterKeys(annotationNames)
+    makeWithAnnotationsAsMdcWithFilter(filter, logFormat)
+  }
+
+  /**
+   * Creates a slf4j logger that puts all the annotations in the MDC context unless excludes by the names that are
+   * defined in `mdcAnnotations` in the MDC context
+   */
+  def makeWithAllAnnotationsAsMdc(
+                                  excludeMdcAnnotations: Set[String] = Set.empty,
```

Comment:
I'm wondering if this should exclude 'throwable' and 'cause' by default?

Reply:
I think it's fine for now